components/core/src/clp/decompression.cpp

```diff
@@ -69,7 +70,7 @@ namespace clp {
             std::unordered_map<string, string> temp_path_to_final_path;
             global_metadata_db->open();
             if (files_to_decompress.empty()) {
-                for (auto archive_ix = std::unique_ptr<GlobalMetadataDB::ArchiveIterator>(global_metadata_db->get_archive_iterator());
+                for (auto archive_ix = std::unique_ptr<GlobalMetadataDB::ArchiveIterator>(global_metadata_db->get_archive_iterator(cEpochTimeMin, cEpochTimeMax));
```

Comment:
Assuming we create a separate iterator for filtering by time, then we shouldn't need to modify this file.

Reply:
Yes, we don't need this. I've reverted this change